parser/src/lexer/tests/list_comprehension.rs

```diff
@@ -0,0 +1,70 @@
+use super::{expect_valid_tokenization, Token};
+
+// LIST COMPREHENSION VALID TOKENIZATION
+// ================================================================================================
+
+#[test]
+fn single_list_comprehension() {
+    let source = "let y = [x for x in x]";
+    let tokens = vec![
```

Comment:
I'm not sure about the term "single". It might be clearer after we switch from "double" to "pair", but I think I'd prefer "item" or "individual". Let's see what others think

Reply:
I think "single" and "pair" work well together.